scripts/setup/debians/opencv/not-installed

```diff
@@ -0,0 +1,235 @@
+usr/bin/opencv_version
```

Comment:
This seems very fragile if a new version for example adds any files. Can we specify files *to* install instead? Is it just the xfeatures2d .so file and a couple others or is there more being installed?

Reply:
we do specify which files to install in libopencv-xfeatures2d.install

Reply:
the not-installed I think it's to be able to ignore the files in the install folder after building that are not to be installed because they are already installed. if we didn't have this the debbuild would fail

Reply:
Can you specify on the directory level and with wildcards instead of listing every file?